crates/common/download/src/download.rs

```diff
@@ -74,15 +73,9 @@ impl Downloader {
             filename.push_str(version.as_str());
         }
 
-        let mut download_target = PathBuf::new().join(&target_dir_path).join(&filename);
-        download_target.set_extension("tmp");
-
```

Comment:
Interesting that this wasn't used anywhere, I remember the file should have been downloaded to a .tmp file and then atomically rewritten, is this logic gone?